app/templates/patch.hbs

```diff
@@ -2,6 +2,10 @@
   <div class="file">
     <div class="meta">
       <span>{{diff.filePath}}</span>
+      <span class="links">
+        <a class="source" href="{{diff.sourceFileUrl}}" title="View generated file at source version">{{diff.sourceVersion}}</a> ...
```

Comment:
Actually, you're right... The git syntax is 3 periods, not an ellipsis character.